.github/workflows/main.yaml

```diff
@@ -5,14 +5,18 @@ jobs:
     steps:
     - name: Checkout
       uses: actions/checkout@v2
-    - name: Build and push latest verifier Docker image
-      uses: docker/build-push-action@v1.1.0
-      with:
-        username: ${{ secrets.DOCKER_USERNAME }}
-        password: ${{ secrets.DOCKER_PASSWORD }}
-        repository: livepeer/verifier
-        tags: latest
-        cache_froms: livepeer/verifier:latest
+    - name: Build base verifier docker image
+      run: docker build -t verifier .
+    - name: Build tests image
+      run: docker build -f Dockerfile-tests -t tests .
+    - name: Run tests
+      run: docker run tests bash -c "mkdir -p ../data/ && wget -c http://storage.googleapis.com/feature_dataset/renditions-nano.tar.gz -O - | tar -xz -C ../data/ && python -m pytest testing/tests"
+    - name: Login to docker
+      run: DOCKER_USERNAME=${{ secrets.DOCKER_USERNAME }} DOCKER_PASSWORD=${{ secrets.DOCKER_PASSWORD }}
```

Comment:
Could these be defined as env variables in the "Push latest verifier image" step instead of setting them in a separate step?

i.e.

```
- name: Push latest verifier image
   run: ...
   env:
        DOCKER_USERNAME=${{ secrets.DOCKER_USERNAME }}
        DOCKER_PASSWORD=${{ secrets.DOCKER_PASSWORD }}
```

Reply:
Done.